typescript/web/src/components/dataset-classes/upsert-class-modal/upsert-class-modal.fixtures.ts

```diff
@@ -0,0 +1,83 @@
+import { pick } from "lodash";
+import { MATCH_ANY_PARAMETERS } from "wildcard-mock-link";
+import { ApolloMocks } from "../../../utils/tests/mock-apollo";
+import { DATASET_LABEL_CLASSES_QUERY } from "./dataset-label-classes.query";
+import { CREATE_LABEL_CLASS_MUTATION } from "./create-label-class.mutation";
+import { LABEL_CLASS_EXISTS_QUERY } from "./label-class-exists.query";
+import { UPDATE_LABEL_CLASS_NAME_MUTATION } from "./update-label-class-name.mutation";
+import {
+  MOCK_DATASET_WITH_CLASSES,
+  MOCK_LABEL_CLASS_SIMPLE,
+} from "../../../utils/tests/data.fixtures";
+
+export const MOCK_LABEL_CLASS_UPDATED_NAME = "My New Class Name";
+
+export const APOLLO_MOCKS: ApolloMocks = {
```

Comment:
This block is very huge.
Maybe we could factorize things a little bit more (and avoid the need for a key-value object).

Reply:
So you think I should put each mock query in its own const?

Reply:
I guess that's the only way to factorize yes. 

I see some pros of doing so:

* We'll have a more robust typing. Since in each mock declaration we would reuse the exact generated apollo types for each query or mutation.
* We can still reference the mocks from outside.
* We won't need `Object.values` to provide the array of mocks.

Do you see some cons?

Reply:
Yeah this is better in several ways I agree, I just wanted to make sure that was what you asked